xbmc/interfaces/builtins/PlayerBuiltins.cpp

```diff
@@ -243,11 +243,6 @@ static int PlayerControl(const std::vector<std::string>& params)
     if( g_application.m_pPlayer->IsPlaying() )
       g_application.m_pPlayer->OnAction(CAction(ACTION_SHOW_VIDEOMENU));
   }
-  else if (paramlow == "record")
-  {
-    if( g_application.m_pPlayer->IsPlaying() && g_application.m_pPlayer->CanRecord())
-      g_application.m_pPlayer->Record(!g_application.m_pPlayer->IsRecording());
-  }
```

Comment:
Isn't this an incompatible JSON-RPC API change as you are removing functionality from the API?

Reply:
player built-ins are for use in skins, i.e. <onclick>PlayerControl(Rewind)</onclick>
JSON-RPC is not supposed to use built-ins

Reply:
Okay, but as PlayerControl(Record) is used by skins (checked with Estuary), this is nothing to to get removed, but to get implemented different (PVR).

Reply:
Record is not a player command, hence it got to be removed here. Either we need some other builtin or observe clicks on REC_BTN in the code.

Reply:
Okay, we already have `CPVRBuildins`. I will add "pvr.recordplayingchannel" there.